boards/shields/max31790/doc/index.rst

```diff
@@ -0,0 +1,29 @@
+.. _max31790_shield:
+
+MAX31790 PWM controller shield
+##############################
+
+Overview
+********
+
+This is a generic shield for the PWM controller MAX31790.
+This controller has an I2C interface and can be operated
+with only SCL, SDA, VCC and GND.
```

Comment:
Can you use the full 80 columns? Or at least the same width on the whole file.

Reply:
done